tools/replay.cpp

```diff
@@ -102,8 +103,19 @@ class Replay {
     static void usage_and_exit( const std::string& errorMessage ) {
       std::cerr << errorMessage
       << std::endl
-      << "Usage: replay <replayfile.csv>"
-      << std::endl;
+      << "Usage: replay <options>\n"
+      << "Common Options:\n"
+      << "    -inputfile filename          CSV file created by Umpire library\n"
```

Comment:
I would prefer using a more robust parser, and having both long and short versions, e.g.: `-i` or `--input`

Reply:
That's fair.  I have used GNU getopt() in the past, but I wasn't sure if it is portable enough.  If you're not suggestion Boost, then do you have another one in mind?